addon/components/o-s-s/private/base-modal.ts

```diff
@@ -40,6 +41,10 @@ export default class BaseModal extends Component<BaseModalArgs> {
       document.body.append(this._parent);
     }
     this._elem = elem;
+
+    window.addEventListener('mousedown', (event) => {
```

Comment:
I see addEventListener but I don't see the removeEventListener to cleanup :)

Reply:
Shouldn't have pushed this in the first place. removing